File: news/src/main/java/material/com/news/holder/NewsHolder.java
package material.com.news.holder;

import android.databinding.DataBindingUtil;
import android.databinding.ViewDataBinding;
import android.support.v7.widget.RecyclerView;
import android.view.View;

import material.com.news.databinding.NewsItemBinding;

/**
 * Created by zjl on 2017/3/27.
 */

public class NewsHolder extends RecyclerView.ViewHolder{
    private ViewDataBinding binding;

    public NewsHolder(View itemView) {
        super(itemView);
        binding = DataBindingUtil.bind(itemView);
    }

    public NewsHolder(NewsItemBinding binding) {
        super(binding.getRoot());
        setBinding(binding);
    }

    public ViewDataBinding getBinding() {
        return binding;
    }

    public void setBinding(ViewDataBinding binding) {
        this.binding = binding;
    }
}

File: news/src/main/java/material/com/news/holder/NewsHolder.kt
package material.com.news.holder

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import android.widget.TextView
import material.com.news.R


/**
 * Created by zjl on 2017/3/27.
 */

class NewsHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    public var img:ImageView
    public var title:TextView
    public var sub_title:TextView

    init {
        img =itemView.findViewById(R.id.news_recyle_img)
        title = itemView.findViewById(R.id.news_recyle_title)
        sub_title = itemView.findViewById(R.id.news_recyle_sub_title)
    }
}
